pytorch_pfn_extras/utils/comparer.py

```diff
@@ -694,18 +695,6 @@ def dump(self, engine: _Engine, dir: str, *args: Any, **kwargs: Any) -> None:
 
         assert isinstance(engine.handler, _ComparableHandler)
         engine.handler = engine.handler._handler
-        summary = {
-            'evaluator': (isinstance(engine, _evaluator.Evaluator)
-                          or getattr(engine, 'evaluator', None) is not None),
-            'trainer': isinstance(engine, _trainer.Trainer),
-            'train_len': getattr(engine, '_train_len', None),
-            'eval_len': engine._eval_len,
-        }
-        if isinstance(engine,_trainer.Trainer):
-            assert engine._manager is not None
-            summary['max_epochs'] = engine._manager.max_epochs
-        with open(f'{dir}/summary', 'w') as f:
-            f.write(json.dumps(summary))
```

Comment:
why these deletions? 😄

Reply:
These deletions are not related to this PR. I will split into another PR.